test/unittests/mongoBackend/mongoNotifyContext_test.cpp

```diff
@@ -980,3 +980,112 @@ TEST(mongoNotifyContextRequest, templateNotificationStdPayload)
 
   utExit();
 }
+
+
+
+/* ****************************************************************************
+*
+* templateNotificationOverrideStdHeader -
```

Comment:
To be removed after closing the end-to-end? Thus, it should be marked with "FIXME: to be removed" or something like that.